plugin/pkg/admission/namespace/autoprovision/admission_test.go

```diff
@@ -81,30 +81,6 @@ func TestAdmissionNamespaceExists(t *testing.T) {
 	}
 }
 
-// TestIgnoreAdmission validates that a request is ignored if its not a create
-func TestIgnoreAdmission(t *testing.T) {
```

Comment:
Ideally, we would have a test that wraps the handler in chainAdmissionHandler and ensures that the request was ignored.  Also it would be good to have tests that Handler returns the expected values since a bad value could cause null pointers.  This probably applies to all the plug-ins, but bugs me most on tests where we were testing that we ignore.